src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -150,3 +152,28 @@ const availableSkus: ISkuOption[] = [
         },
     },
 ];
+
+// Official storage sizes are 32768, 65536, 131072, 262144, 524288, 1048576, 2097152, 4194304, 8388608, 16777216
+const flexibleServerSkus: ISkuOption[] = [
+    {
+        label: "B1ms",
```

Comment:
Is there any info you could add to the description that would account for why "ms" and "s" are included in the labels? Or is this common SKU naming that people will be familiar with? It just looks a little strange to me
![Screen Shot 2021-06-22 at 12 06 46 PM](https://user-images.githubusercontent.com/22795803/122984870-59a9a080-d352-11eb-953c-cfc1cccd61ef.png)

Reply:
This is how they're documented in the official docs https://docs.microsoft.com/en-au/azure/postgresql/flexible-server/concepts-compute-storage#compute-tiers-vcores-and-server-types

I don't know what m or s means.